https://gitlab.gnome.org/GNOME/gnome-bluetooth/-/commit/755fd758f866d3a3f7ca482942beee749f13a91e
https://bugs.gentoo.org/831430

From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 7 Jan 2022 12:51:22 +0100
Subject: [PATCH] build: Fix build for newer versions of meson

sendto/meson.build:24:5: ERROR: Function does not take positional arguments.
--- a/sendto/meson.build
+++ b/sendto/meson.build
@@ -22,7 +22,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file (
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
GitLab
